README.md

```diff
@@ -100,7 +107,26 @@ Each lion should have...
 Create a new lion instance with the name "Simba".
 
 ```ruby
-# code here
+class Animal
+  attr_accessor :name, :greet
+  def initialize (name, greet)
+    @name = name
+    @greet = greet
+  end
+end
+
+animal = Animal.new("Pumba", nil)
+# Ceating Pumba works. Getting and "Uninitilized constant error for Lion class"  
+
+class Person < Lion
+    attr_accessor :name, :greet
+    def initialize (name, greet)
+    @name = name
+    @greet = greet
+  end
+end
+
+lion = Lion.new("Simba")
```

Comment:
Want to define a greet method on animal and then for lion to look something like this:
```ruby
class Lion < Animal
  @@pack = []
  def initialize (name)
    @name = name
    @king = name == "Simba"
    @@pack << self
  end
end
```